repo_policy_compliance/github_client.py

```diff
@@ -103,7 +103,12 @@ def get_collaborators(
     """
     collaborators_url = repository.collaborators_url.replace("{/collaborator}", "")
     default_query = dict(parse.parse_qsl(parse.urlparse(collaborators_url).query))
-    query: dict[str, str] = {**default_query, "permission": permission, "affiliation": affiliation}
+    query: dict[str, str] = {
```

Comment:
Perhaps for a follow up PR, it still looks like we need to get full results for the push logins since that is used to authorize the comment for approving a run for an external fork. In that case we probably do need to retrieve all push logins since we have to compare the set of users that commented the authorization string to the set of allowed users in which case it would be expensive to check each one individually. In most cases I would expect there only to be 1 valid authorization comment but probably better for us to handle the case where there are a lot

Reply:
Created an issue #925 for follow up!